lib/sms.js

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const libpn = require('google-libphonenumber')
+const Nexmo = require('nexmo')
+const P = require('bluebird')
+const SENDER_IDS = new Map(require('./sms-sender-ids.json'))
+
+module.exports = (log, translator, templates, config) => {
+  const nexmo = new Nexmo({
+    apiKey: config.apiKey,
+    apiSecret: config.apiSecret
+  })
+  const sendSms = P.promisify(nexmo.message.sendSms, { context: nexmo.message })
```

Comment:
This may be pre-mature, any value in passing in the `sendSms` function so that we can swap out providers if need be, e.g., if we decide to use Basket or Amazon SNS instead of Nexmo? Basket may duplicate a lot of the functionality here, I'm not sure how it works.